pkg/parser/extract.go

```diff
@@ -83,7 +85,7 @@ func GetGV(f flag.Flags) ([]types.GroupVersions, error) {
 	return groupVersions, nil
 }
 
-func GetGVKs(ctx *genall.GenerationContext, inputDir string, groupVersions []types.GroupVersions) (map[Group]map[types.PackageVersion][]Kind, error) {
+func GetGVKs(ctx *genall.GenerationContext, inputDir string, groupVersions []types.GroupVersions, requiredVerbs []string) (map[Group]map[types.PackageVersion][]Kind, error) {
```

Comment:
(no action required) if you make requiredVerbs a sets.String, you can invoke HasAll on it

Reply:
where is `sets` from? my editor grabs `"sigs.k8s.io/kustomize/kyaml/sets"`, which I assume isn't the right one

Reply:
hmm gave it a go but it ended up being a little less clear to me than just using the map[string]bool and []string